src/api/operations.ts

```diff
@@ -417,14 +428,11 @@ export const drawEllipse = (options: {
           rotate: options.rotate ?? degrees(0),
         })),
 
-    // prettier-ignore
-    options.color && options.borderWidth ? fillAndStroke()
-  : options.color                      ? fill()
```

Comment:
I am not sure if this is an issue or not, but this did not previously use the `fillRule`, and now it does.